1.8/usage/cli/multi-cluster-cli.md

```diff
@@ -0,0 +1,33 @@
+---
+post_title: Connecting to Multiple Clusters
+nav_title: Multiple Clusters  
+menu_order: 3.1
+---
+
+You can connect the CLI to multiple DC/OS clusters at the same time by using the `DCOS_CONFIG` environment variable. With this variable you can specify a terminal-specific CLI configurations. 
+
+This variable only affects the terminal you are currently connected to. After you end the terminal session, the configuration is removed and your default configuration remains.
+
+**Recommendation:** Make a back up your default CLI configuration file (`dcos.toml`) before connecting to multiple clusters. By default, this file is located in `/<home-directory>/.dcos/dcos.toml`. You can use this file as a template for additional configurations.
+
+1.  Create a cluster configuration file and save to a unique location (e.g. `/dcos-config/dcos-alice.toml`). The file type must be `.toml`.
```

Comment:
great catch - this is news to me. i used to avoid `e.g.`, but have given up that fight. back in the IBM days this was identified as bad for translation purposes.